File: Linguagem Java/Soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class Soma_vetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int n = sc.nextInt();
		
		double[] vet = new double[n];
		
		for(int i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
		}
		
		System.out.print("Valores = ");
		for(int i = 0; i < n; i++) {
			System.out.print(vet[i] + " ");
		}
		
		int quant = 0;
		double soma = 0;
		for(int i = 0; i < n; i++) {
			soma = soma + vet[i];
			quant++;
		}
		System.out.print("\n");
		System.out.println("Soma = " + soma);
		System.out.println("Media = "+ (double)soma/quant);
		
	}

}

File: Linguagem Python/EXs - Vetores PY/soma_vetor.py
n = int(input("Quantos numeros voce vai digitar? "))
vet: [float] = [0 for x in range(n)]
for i in range(0, n):
    vet[i] = float(input("Digite um numero: "))

print("Valores = ", end= " ")
for i in range(0, n):
    print(f"{vet[i]} ", end=" ")

soma = 0
quant = 0
for i in range(0, n):
    soma = soma + vet[i]
    quant = quant + 1
print()
print(f"Soma = {soma:.2f}")
print(f"Media = {soma/quant:.2f}")
